config/clusters/meom-ige/cluster.yaml

```diff
@@ -38,3 +38,13 @@ hubs:
       # that you intend for these files to be applied in this order.
       - common.values.yaml
       - prod.values.yaml
+  - name: drakkar-demo
+    display_name: "2023 DRAKKAR Ocean Modelling Workshop"
+    domain: drakkar-demo.meom-ige.2i2c.cloud
+    helm_chart: basehub
+    auth0:
+      # connection update? Also ensure the basehub Helm chart is provided a
+      # matching value for jupyterhub.custom.2i2c.add_staff_user_ids_of_type!
+      connection: github
```

Comment:
@consideRatio, I believe we generally wish to move away from using Auth0. Though, I don't think there's a clear timeline about when/if to remove it from our infrastructure.

For now, I believe you could use the GitHubOAuthenticator directly.

Reply:
I think this is addressed by 00a928d

Reply:
Great! Thank you @consideRatio